design/protected-apps-view.md

```diff
@@ -0,0 +1,917 @@
+# Design Document: Protected Applications Unified View
+## OpenShift Data Foundation - Disaster Recovery
+
+---
+
+## Document Information
+
+**Version:** 1.0  
+**Date:** November 2025  
+**Status:** Proposal  
+**Epic:** Enhance Protected Applications page to support managed and discovered applications
+
+---
+
+## 1. Goal and Motivation
+
+### 1.1 The Challenge We're Solving
+
+Currently, our Protected Applications page displays only discovered applications, which are applications protected at the namespace level through DRPlacementControl (DRPC) resources in the `openshift-dr-ops` namespace. However, Red Hat Advanced Cluster Management (ACM) supports two additional application types that can also be protected with disaster recovery capabilities: ApplicationSets (deployed via GitOps/ArgoCD) and Subscriptions (the legacy ACM application deployment model). 
```

Comment:
Considering this is a new design proposal and "Subscriptions" application type is not supported/legacy now, do we still want to support that?